detox/index.d.ts

```diff
@@ -1445,6 +1462,149 @@ declare global {
             };
         }
 
+        // Element Attributes Shared Among iOS and Android
+        interface ElementAttributes {
```

Comment:
Just to point out, it seems like there are more attributes that we can provide with this API (see https://reactnative.dev/docs/accessibility), like `accessibilityHint`, or `accessibilityState`..

Reply:
Not sure, @asafkorem.

I intercepted what the iOS side returns and I see no traces of the mentioned properties:

```json
{
  "elementSafeBounds": {
    "y": 0,
    "x": 0,
    "width": 426,
    "height": 149.99996948242188
  },
  "hittable": true,
  "frame": {
    "y": 469.3333435058594,
    "x": 1,
    "width": 426,
    "height": 149.99996948242188
  },
  "normalizedActivationPoint": {
    "x": 0.5,
    "y": 0.5
  },
  "visible": true,
  "elementBounds": {
    "y": 0,
    "x": 0,
    "width": 426,
    "height": 149.99996948242188
  },
  "enabled": true,
  "layer": "<CALayer: 0x60000281b8a0>",
  "safeAreaInsets": {
    "right": 0,
    "top": 0,
    "left": 0,
    "bottom": 0
  },
  "className": "RCTScrollContentView",
  "activationPoint": {
    "x": 213,
    "y": 74.99998474121094
  },
  "elementFrame": {
    "y": 0,
    "x": 0,
    "width": 426,
    "height": 149.99996948242188
  }
}
```

If you meant we can extract them potentially, then it is out of the scope of this PR, definitely. Here we are just adding typings and making sure that they are totally correct.

Reply:
Sure, out of this PR scope. I'm talking about a change that requires both Android and iOS implementations.

Reply:
https://github.com/wix/Detox/issues/3116